SuperteamDAO/earn
# fix:#478 added upload image feature in the listing description box

## This PR resolves the issue mentioned in #478 

## Where should the reviewer start?
Start by reviewing the changes made in the DescriptionBuilder component file (DescriptionBuilder.tsx). Look for the modifications in the event handling and editor interaction sections.

## How should this be manually tested?

1. Open the Listing Details box component.
2. Trigger the image input/upload event by selecting a file
3. Verify that the file is correctly extracted and processed.

## Any background context you want to provide?
I also implemented the Heading Feature which I mentioned in this comment: [LINK](https://github.com/SuperteamDAO/earn/issues/478#issuecomment-2014807093)

## What are the relevant issues?
Add support for images in the listing description #478

## Working Demo
https://github.com/SuperteamDAO/earn/assets/40737575/eab9dc0d-3e66-4995-a222-06714d66791b
